app/serializers/document_serializer.rb

```diff
@@ -1,24 +1,27 @@
 # frozen_string_literal: true
 
-class DocumentSerializer < ActiveModel::Serializer
+class DocumentSerializer
+  include JSONAPI::Serializer
   include Rails.application.routes.url_helpers
   include ActionView::Helpers::NumberHelper
 
-  attributes :id, :url, :filename, :filesize, :content_type
+  set_type :documents
 
-  def url
+  attributes :id
```

Comment:
I don't think we want the id in the attributes?

Reply:
I wondered about this, it's included in most but not all of the existing serializers - although it shouldn't be needed according to the JSONAPI spec. I'll try removing `:id` from all of the serializers and see what happens 🤔